src/views/connection.vue

```diff
@@ -1,23 +1,26 @@
 <script>
-import { mapActions, mapState } from 'vuex'
+import { mapActions, mapState } from "vuex";
```

Comment:
We use singlequotes instead of double quotes. It would be great if you could remove these lines from the PR. We'll introduce a .prettierrc file later — sorry for the inconvenience.

Reply:
I've noticed this and changed my editor settings accordingly but didn't noticed that gitHook under package.json was changing single quotes to double quotes while committing. It should be fixed now.